bin/disco_ssh.py

```diff
@@ -133,16 +135,31 @@ def build_ssh_cmd(self, ips):
         """
         Given a list of ip addresses, build an ssh command to tunnel through n-1 ips to reach the n-th ip
         """
-        command = " ".join([
-            "ssh -At {} {}".format(SSH_OPTIONS, ip)
-            for ip in ips])
+        if self.tunnel:
+            lport, rport = self.tunnel.split(":")
+            host = self.args["<host>"]
```

Comment:
Would there be any utility in providing a second usage?
```
Usage:
    disco_ssh.py [--env ENV --debug --first] <host>
    disco_ssh.py [options] tunnel <target>

Variables:
     <host>                 A hostname/hostclass/instance substring
     <target>               The host to tunnel to.

Options:
     -h --help              Show this screen
     --debug                Log in debug level
     --env ENV              Environment to operate in
     --first                In case of multiple matching instances, connect to the first instead of failing
     --ports LPORT:RPORT   Establish a secure connection between a local port and a remote port on the host
```